modules/sdk/project-templates-mvc-war-portlet/src/main/resources/archetype-resources/src/main/webapp/view.jsp

```diff
@@ -0,0 +1,5 @@
+<%@ include file="/init.jsp" %>
+
+<p>
+	<b>Hello from <%= renderRequest.getAttribute("helloWorld") %></b>
```

Comment:
Lets have this use the content/Language.properties like the other MVC portlet examples do.